src/EditorFeatures/CSharpTest/QuickInfo/SyntacticQuickInfoSourceTests.cs

```diff
@@ -294,10 +289,18 @@ private IQuickInfoProvider CreateProvider(TestWorkspace workspace)
             string expectedDocumentationComment = null)
         {
             var provider = CreateProvider(workspace);
-            var state = await provider.GetItemAsync(document, position, cancellationToken: CancellationToken.None);
-            Assert.NotNull(state);
-
-            var viewHostingControl = (ViewHostingControl)((ElisionBufferDeferredContent)state.Content).Create();
+            var info = await provider.GetQuickInfoAsync(document, position, cancellationToken: CancellationToken.None);
+            Assert.NotNull(info);
+
+            Assert.NotEqual(0, info.RelatedSpans.Length);
+            var tabSize = document.Options.GetOption(Microsoft.CodeAnalysis.Formatting.FormattingOptions.TabSize, document.Project.Language);
+            var text = await document.GetTextAsync().ConfigureAwait(false);
```

Comment:
tests do not use ConfigureAwait.  Everything in tests is implicitly ConfigureAwait(true).